inputs/train_ferc1_eia.csv

```diff
@@ -1,805 +1,2752 @@
-record_id_eia,record_id_ferc1,signature_1,notes
```

Comment:
do you know why the diff is the whole file? is there some sorting going on before they get dumped back into this file? It would probably be good to try to minimize the diffs so we can actually see what gets added/removed each time.

but also the sheer volume of additions is grand! probably why the f-score went up so much with these new bbs!!

Reply:
I think it's because the current system over-writes the file every time. There is a log output that shows you how many new records were added, but yeah we can probably "add to" the excel doc instead of slurping it out of excel adding new stuff in pandas and then outputting the whole new thing to excel.

Reply:
I've had weird luck with getting the CSV's to diff properly - sometimes a full overwrite works fine and other times it fails. I'm sure there is some magic that we don't understand, but don't worry about it for now.